### PR TITLE
Fix secret key import error

### PR DESCRIPTION
**Pull request by OKendigelyan**
## Proposed changes

[Task link](https://app.asana.com/0/1205770721172203/1207311817583468/f)

## Types of changes

- [x] Bugfix
- [ ] New feature
- [ ] Refactor
- [ ] Breaking change
- [x] UI fix

## Screenshots

Add the screenshots of how the app used to look like and how it looks now

| Before | Now |
| ------ | --- |
|image|image|

## Checklist

- [x] Tests that prove my fix is effective or that my feature works have been added
- [ ] Documentation has been added (if appropriate)
- [x] Screenshots are added (if any UI changes have been made)
- [ ] All TODOs have a corresponding task created (and the link is attached to it)

### CODE REVIEW

File: src/components/Onboarding/restoreSecretKey/RestoreSecretKey.tsx [hunk 1]
```diff
@@ -68,44 +64,43 @@ export const RestoreSecretKey = ({ goToStep }: { goToStep: (step: OnboardingStep
...
           </FormControl>
-        )}
 
-        <Button width="100%" marginTop="32px" isDisabled={!isValid} size="lg" type="submit">
-          Continue
-        </Button>
-      </form>
+          {isEncrypted && (
+            <FormControl marginTop="20px" isInvalid={!!errors.password}>
+              <PasswordInput data-testid="password" inputName="password" minLength={0} />
```
Comment (asiia-trilitech) [inline src/components/Onboarding/restoreSecretKey/RestoreSecretKey.tsx:87]: Just wanted to check the reasoning for `minLength={0}`

I think we have `minLength=8` in Umami 🤔 
However, for random secret keys it makes sense to me to allow different values

**Review by serjonya-trili — APPROVED** [review]
LGTM 👍

File: src/components/Onboarding/restoreSecretKey/RestoreSecretKey.test.tsx
Comment (asiia-trilitech) [inline src/components/Onboarding/restoreSecretKey/RestoreSecretKey.test.tsx:1]: @OKendigelyan could you please add a test for the error being displayed correctly?

And another one for having passwords with length less than 8?

**Review by asiia-trilitech — APPROVED** [review]
Thanks Oleg - the code looks great!

File: src/components/Onboarding/restoreSecretKey/RestoreSecretKey.test.tsx
Comment (OKendigelyan) [reply src/components/Onboarding/restoreSecretKey/RestoreSecretKey.test.tsx:1]: @asiia-trilitech if I understood you correctly, there is already a test for correct error display. 
This implementation assumes any password, so the test for this case already exists - _'shows an error if the password is invalid'_.

**Pull request merged**